docs/development.md

```diff
@@ -46,19 +47,91 @@ centos7-devel:
         "
 ```
 
+Next, you may want to edit the Vagrantfile to forward some ports from the guest to the host:
+
+```
+Vagrant.configure("2") do |config|
+  config.vm.network :forwarded_port, guest: 443, host: 8443, host_ip: "0.0.0.0"
+  config.vm.network :forwarded_port, guest: 3808, host: 3808, host_ip: "0.0.0.0"
```

Comment:
Yeah, if you run 'foreman start' instead of running via rails with the webpack dev server turned on you need 3808. Which, if you want to use the HTTPS reverse proxy, was the only way I could get it to work.